packages/contracts-bedrock/deploy-config/sepolia.json

```diff
@@ -0,0 +1,41 @@
+{
+  "finalSystemOwner": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+  "portalGuardian": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+  "l1StartingBlockTag": "",
```

Comment:
Are we picking this explicitly at a later point, or do we just leave it empty?

Reply:
We need to explicitly pick this at a later point, after contract deployment
https://github.com/ethereum-optimism/optimism/blob/9af277e9d0ac7cf89d8409234241869241c1b0a3/op-node/cmd/genesis/cmd.go#L136